### PR TITLE
change default settings file

### PR DESCRIPTION
**Pull request by EDIflyer**
**Description**
other documentation uses `/database.db` so change from `/database/filebrowser.db` for consistency (had lots of file not found issues before when trying to use default config!)
(see PR in docs - https://github.com/filebrowser/docs/pull/47)

- [x] DO make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your master!
- [x] DO make sure you are making a pull request against the **master branch** (left side). Also you should start *your branch* off *our master*.
- [x] DO make sure that File Browser can be successfully built. See [builds](https://github.com/filebrowser/community/blob/master/builds.md) and [development](https://github.com/filebrowser/community/blob/master/development.md).
- [x] DO make sure that related issues are opened in other repositories. I.e., the frontend, caddy plugins or the web page need to be updated accordingly.
- [x] AVOID breaking the continuous integration build.

**Further comments**
Ran into lots of permission issues when setting up in docker, then realised filenames were different between config and example docker script.

### CODE REVIEW

**Comment by EDIflyer** [comment]
Bump - still awaiting review

**Comment by EDIflyer** [comment]
Bump

**Comment by EDIflyer** [comment]
Bump to remove stale label

File: docker/root/defaults/settings.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "baseURL": "",
   "address": "",
   "log": "stdout",
-  "database": "/database/filebrowser.db",
```
Comment (o1egl) [inline docker/root/defaults/settings.json:6]: This settings file is related to the linuxserver based container and the path of the file is set correctly

Comment (EDIflyer) [reply docker/root/defaults/settings.json:6]: It should probably be changed elsewhere then? The documentation seemed to vary between `/database/filebrowser.db` and `/database.db`? For example what caught me out was the Alpine example at https://filebrowser.org/installation#docker uses `/database.db` but then the text below to the sample config file at https://github.com/filebrowser/filebrowser/blob/master/docker/root/defaults/settings.json is then `/database/filebrowser.db`? Perhaps it just needs some text added to make it clear you need to change the sample config file to match? (I'll confess when I first set things up I had a mismatch and it took me a while to work out what was going on!).  I think if you don't have a config file it seems to default to expect `/database.db`?

**Comment by EDIflyer** [comment]
Adding comment due to `stale` label - As mentioned above I think still needs some tweaks to create consistency?

File: docker/root/defaults/settings.json (hunk 1 above)
Comment (o1egl) [reply docker/root/defaults/settings.json:6]: No, the default docker image https://github.com/filebrowser/filebrowser/blob/master/Dockerfile uses the https://github.com/filebrowser/filebrowser/blob/master/docker_config.json for config. The file you are referring to is used in the linuxserver based image. But I agree that the docs need to be changed. Feel free to make a [PR](https://github.com/filebrowser/docs/pulls)

Comment (EDIflyer) [reply docker/root/defaults/settings.json:6]: OK I'll try to look hopefully this weekend (PS just synced my fork but it seemed to appear as ready to merge but not changed yet - will post when it is!)

**Comment by EDIflyer** [comment]
Going to work on this (commenting to remove stale label)

File: docker/root/defaults/settings.json (hunk 1 above)
Comment (EDIflyer) [reply docker/root/defaults/settings.json:6]: Sorry for the delay - got round to it in https://github.com/filebrowser/docs/pull/52 by tweaking the docs very slightly

**Comment by o1egl** [comment]
@EDIflyer I merged your docs changes. So I think this PR can be closed.

**Pull request closed without merging**